sensor_msgs/Image[] objects
---
GenderAndAgeList gender_and_age_response
